learn/src/main/java/org/navgurukul/learn/ui/learn/AssessmentFragment.kt

```diff
@@ -101,38 +97,28 @@ class AssessmentFragment : Fragment() {
                 is AssessmentFragmentViewModel.AssessmentFragmentViewEvents.ShowToast -> toast(it.toastText)
                 is AssessmentFragmentViewModel.AssessmentFragmentViewEvents.ShowCorrectOutput -> {
                     isContentRvClickable = false
-//                    isContentRvClickableMultiple = false
```

Comment:
We can remove this comment as well @aayushitiwari03

Reply:
Sure, I'll check the code and remove the unwanted code.